iati/validator.py

```diff
@@ -136,10 +137,33 @@ def contains_error_of_type(self, err_type):
             bool: Whether there is an error or warning with the specified type within the log.
 
         """
-        errors_with_type = [err for err in self._values if err.base_exception == err_type]
+        errors_with_type = self.get_errors_or_warning_by_type(err_type)
 
         return len(errors_with_type) > 0
 
+    def contains_errors(self):
+        """Determine whether there are errors contained within the ErrorLog.
+
+        Note:
+            The error log may contain warnings, or may be empty.
+
+        Returns:
+            bool: Whether there are errors within this error log.
+
+        """
+        return len(self.get_errors()) > 0
```

Comment:
Something like `contains_error_of_type` uses a temp variable. Should be consistent.

Reply:
Done in a4412360192de91b95c4a7817b893f53df5032df